src/libshadertrap/include/libshadertrap/executor.h

```diff
@@ -16,7 +16,12 @@
 #define LIBSHADERTRAP_EXECUTOR_H
 
 #define GL_GLES_PROTOTYPES 0
+#ifdef SHADERTRAP_DEQP
```

Comment:
Actually, I think move GL_GLES_PROTOTYPES 0 into the #else block. Same thing for gl_functions.h.

Reply:
done